CHANGELOG.md

```diff
@@ -7,10 +7,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.6.0] - 2019-09-28
```

Comment:
@aburgel is right. The date seems off ;-)

Reply:
Oooops. Yeah.